Changelog.md

```diff
@@ -4,7 +4,7 @@ Language Features:
 
 
 Compiler Features:
-
```

Comment:
We usually keep two empty lines here, so if you delete it, we'll need to add one when sorting the changelog anyway ;)